### PR TITLE
write topic, move exporter samples to topics

### PR DESCRIPTION
**Pull request by hanastasov**
Write topic for Configuring igGridExcelExporter
Move documentation samples to topics

### CODE REVIEW

**Review by tiliev — APPROVED** [review]

File: topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+
+### <a id="gridfeatureoptions_configure"></a>Configure the `gridFeatureOptions` settings
+
+Data and layout manipulations caused by the `Filtering`, `Hiding`, `Paging`, `Sorting`, `Summaries` and `Column Fixing` features of the `igGrid` can be reflected in the exported Excel Worksheet. The setting for each feature is passed within the [`gridFeatureOptions`](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions) object.
+All of these are enumerations and the `sorting`, `summaries` and `columnfixing` have only options - `"none"` or `"applied"`.  
+
+|Setting | Property | Description
+| -------------------| ------------------- |----------- 
+Configure Sorting | [sorting](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions.sorting) | Accepts values `"none"` or `"applied"`. When `"none"` Sorting is not applied in the Excel. When `"applied"` Sorting is applied in the Excel Worksheet.
```
Comment (mpavlinov) [inline topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md]: Please update the reference table for the options that you talk about in this section. Currently it's hard to keep track of what options you talk about and what are the possible values that the developer can set for them.

File: topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        columnfixing: "applied",
+        // summaries will not be calculated in the worksheet
+        summaries: "none",
+        // Sorted `igGrid` columns will be sorted in the excel worksheet
+        sorting: "applied"
+    }
+});
+```
+
+Different options are available for configuring the `filtering`, `paging` and `hiding`. For example, we can opt to work with the visible `igGrid` columns only and do not export the hidden ones by setting `hiding: "visibleColumnsOnly"`. The other possible options for hiding are still `"none"` and `"applied"`. Setting `paging` and `filtering` is tricky as both features manipulate the data and different combinations will bring different results. For `paging` we can set either `"currentPage"` that will export only the current `igGrid` page, or `"allRows"` correspondingly. `filtering` can take values `"none"`, `"applied"`, or `"filteredRowsOnly"`. The difference between them - setting `"applied"` will make the `igGridExcelExporter` export all data and apply the same filter as applied in the `igGrid`, while setting `"filteredRowsOnly"` will export only the filtered records.
```
Comment (mpavlinov) [inline topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md]: You can put this information into the table above. Keep in mind that this topic is not to repeat the information from the API docs, but to add more value for the reader and actually describe actual end user scenarios. You can see how other "Configuration" topics are written. Most of them focus on user stories.

File: topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+|metadata|
+-->
+
+# Configuring igGridExcelExporter
+
+## Topic Overview
+
+### Purpose
+
+The `igGridExcelExporter`™ provides options that allow any data and layout manipulations caused by the `igGrid` features to be reflected in the Excel Worksheet, as well as to configure styles to be applied, columns to be skipped during exporting, setting file and worksheet names, etc.
```
Comment (mpavlinov) [inline topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md]: Excel is a name of a product and always should be capitalized.

File: topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+-   [Display an overlay while exporting](#exporting_overlay)
+
+### Required background
+- [igGridExcelExporter Overview](iggridexcelexporter-overview.html "igGridExcelExporter Overview") - General information on the `igGridExcelExporter` control.
+
+
+### <a id="gridfeatureoptions_configure"></a>Configure the `gridFeatureOptions` settings
+
+Data and layout manipulations caused by the `Filtering`, `Hiding`, `Paging`, `Sorting`, `Summaries` and `Column Fixing` features of the `igGrid` can be reflected in the exported Excel Worksheet. The setting for each feature is passed within the [`gridFeatureOptions`](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions) object.
+All of these are enumerations and the `sorting`, `summaries` and `columnfixing` have only options - `"none"` or `"applied"`.
```
Comment (mpavlinov) [inline topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md]: The option values that are strings should be enclosed in quotation marks.

File: topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+### <a id="exporting_overlay"></a>Display an overlay while exporting
+
+During exporting the `igGrid` UI is not available for interactions so you may want to place a modal overlay above. It will show in the beginning of exporting and hide in the end. To implement this the `exportStarting` and `success` callbacks are used:
+
+```javascript
+$.ig.GridExcelExporter.exportGrid($("#grid1"), {},
+{
+    exportStarting: function (e, args) {
+        showOverlay(args.grid, exportingOverlay);
```
Comment (mpavlinov) [inline topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md:162]: As a reader I'm interested what is the definition of the `showOverlay` and `hideOverlay` functions. Please add their definitions to the code sample or better add a live sample.

**Review by mpavlinov — CHANGES_REQUESTED** [review]
I edited all of the topics with my changes. However "Configuring igGridExcelExporter" still needs work.

File: topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md (hunk 5 above)
Comment (hanastasov) [reply topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md:162]: The content of `showOverlay` and `hideOverlay` is available in the live sample, which is in the end of document and I would prefer keeping the code sample more readable as it is now.

**Comment by hanastasov** [comment]
Requested changes are introduced to the topic.

**Review by mpavlinov — APPROVED** [review]

**Pull request merged**